src/patternfly/components/OverflowMenu/docs/overflow-menu-group-types.md

```diff
@@ -0,0 +1,8 @@
+The action group consists a primary and secondary action. Any additional actions are part of the overflow control dropdown.
+
+### Overflow menu group types
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-overflow-menu__group` | `<div>` | Initiates the overflow menu component group. |
+| `.pf-m-button-group` | `.pf-c-overflow-menu__group` | Modifies overflow menu group spacing. Spacer value is set to `var(--pf-c-overflow-menu__group--m-button-group--spacer)`. Child spacer value is set to `var(--pf-c-overflow-menu__group--m-button-group--space-items)`. |
```

Comment:
What happens if there is a button and a button icon in a group? I feel like these modifiers are very prescriptive so could we name them differently but then show buttons/icons being used in them?

Reply:
Groups can be nested. So it would look like this.
```
.pf-c-overflow-menu
  .pf-c-overflow-menu__group
    .pf-c-overflow-menu__group.pf-m-icon-button-group
      .pf-c-overflow-menu__item <icon button>
      .pf-c-overflow-menu__item <icon button>
    .pf-c-overflow-menu__group.pf-m-button-group
      .pf-c-overflow-menu__item <button>
      .pf-c-overflow-menu__item <button>
```
or 
```
.pf-c-overflow-menu
  .pf-c-overflow-menu__item <button>
  .pf-c-overflow-menu__item <icon button>
```
From there, `.pf-m-spacer-{size}` would handle any more specific spacing.